code/modules/organs/external/head.dm

```diff
@@ -27,6 +27,15 @@
 	var/forehead_graffiti
 	var/graffiti_style
 
+/obj/item/organ/external/head/emp_act(severity)
+	if(BP_IS_ROBOTIC(src))
+		var/obj/item/organ/internal/mmi_holder/FBP_brain = owner.internal_organs_by_name[BP_BRAIN]
+		if(istype(FBP_brain))
+			FBP_brain.stored_mmi.forceMove(get_turf(owner))
+			owner.mind.transfer_to(FBP_brain.stored_mmi.brainmob)
+			FBP_brain.stored_mmi.visible_message(SPAN_DANGER("Alarm. Auto eject module of [owner]'s head activated due to unforeseen consequences. Man-Machine Interface ejected successfully."), SPAN_NOTICE("The Man-Machine Interface appears from [owner]'s head"))
```

Comment:
Не особо понял, что написано, но я бы воткнул что-то вроде
SPAN("danger", "You see a bright flash as you get catapulted out of your body. You feel disoriented, which must be normal since you're but a brain now.", 
SPAN("notice", "[owner]'s head ejects an MMI!")

Reply:
Да, примерно это я и хотел, исправлю на этот вариант

Reply:
> you're but a brain now

Разве не ```you're a brain now```?

Reply:
you're _but_ a brain - "ты _всего-лишь_ (жалкий, застрявший в банке, не могущий даже в suicide/ghost, не говоря уже о респавне и даже дедчатике) мозг"

Reply:
Может лучше ```you're just a brain```?

Reply:
But - всего-лишь, just - просто. Выбирай шо хош, можно и просто юра брейн.

Reply:
Никогда не слышал, что бы but использовали в контексте всего-лишь, ну хоть что-то новое узнал.

Reply:
Сделал, закрываю

Reply:
![image](https://user-images.githubusercontent.com/45202681/96643324-38b72880-1349-11eb-8d25-e3e90eacb9d3.png)
Классику смотреть надо